SendCustomerData_to_TalkSoft.feature

```diff
@@ -1,100 +0,0 @@
-Feature: Provide Customer Data and Client appointment records to Talksoft
-Ability for RevSpring to send Customer Data and Client appointment records to Talksoft
-
-Scenario: RemindMe in PST time only 
-Given that Client opted for "PST' in "timezonename' field
-And I opt-in to receive email communication in PST timezone
-When I schedule an appointment
-Then I should receive email/Text/IVR communication in PST time only
-
-Scenario: RemindMe should send IVR message  only 
-Given that I opt-in for  'Y' in 'Phone Allowed' field and 'N' in 'Text allowed' and 'N' in 'Email allowed' field.
-When I Schedule an appointment
-Then I should get an IVR message
```

Comment:
I think that this is not correct.

I think a scenario is missing indicating that IWS to handle scheduling and not TalkSoft.